src/config.rs

```diff
@@ -50,11 +50,10 @@ pub struct TlsConfig {
 }
 
 /// Generation service provider
-#[cfg_attr(test, derive(Default))]
-#[derive(Clone, Copy, Debug, Deserialize)]
+#[derive(Clone, Copy, Debug, Default, Deserialize)]
```

Comment:
I am afraid making connections default for non unit test cases can create weird errors later on if some fields get automatic unexpected defaults 🤔 Is there a reason to make default for everything or  this is fkr cleanup?

Reply:
I will revert this